### ISSUE TITLE
Elements transition: How to get the elements in the new document?

### ISSUE THREAD

**Issue by verlok** [vmpstr/shared-element-transitions#21]
I am reading the readme and I’ve found this portion of code for element transition:

```js 
function handleTransition() {
  document.documentTransition.prepare({
    rootTransition: "reveal-left",
    duration: 300,
    sharedElements: [e1, e2, e3]
  }).then(() => {
    changeBodyBackground();
    document.documentTransition.start([newE1, newE2, newE3]).then(
      () => console.log("transition finished"));
  });
}
```

What it’s not clear to me is: 
Using which `document` object should we “querySelect” the elements, for both `[e1, e2, e3]` and `[newE1, newE2, newE3]`?

**Comment by vmpstr** [comment]
Yes, in the single-page app case. Specifically, the flow work as follows:

* You call `prepare()` which returns a promise
* The promise resolves, and you change your page in any way that you want. This includes adding new elements, such as `newE1`, `newE2`, and `newE3` in the example. Note that at this point, the elements are part of the DOM so you can querySelect them, but they are not yet visually on screen since they were just added as part of this promise resolution
* You call `start()` which is a signal to the browser that instead of simply displaying the new page (with `newE1` etc), it should do an animation from the old prepared state to the new state

I'll update the README to try and make this more clear. The intent is that `changeBodyBackground()` call can also insert `newE1` etc to the page.

Please let me know if that makes sense.

**Comment by verlok** [comment]
I get it now. 
I’m more interested in the multiple page application case. 
Do you have any example for that case?
Btw I wasn’t able to find any working demo of that case. Do you know if there are any?

**Comment by vmpstr** [comment]
We don't current have any examples of the multiple page app case. The difficulty there is, as you say, identifying the elements on the destination page, which isn't as easy as the SPA case.

We're still in the early stages, but we're working on adding multipage support

**Comment by khushalsagar** [comment]
This seems like a duplicate of #16.
